src/engraving/rw/read400/tread.cpp

```diff
@@ -3772,7 +3772,7 @@ bool TRead::readProperties(Staff* s, XmlReader& e, ReadContext& ctx)
     } else if (tag == "hideSystemBarLine") {
         s->setHideSystemBarLine(e.readInt());
     } else if (tag == "mergeMatchingRests") {
-        s->setMergeMatchingRests(e.readInt());
+        s->setMergeMatchingRests(AutoOnOff(static_cast<char>(e.readInt())));
```

Comment:
In the same spirit as the comment in twrite:
```suggestion
        s->setMergeMatchingRests(e.readInt() ? AutoOnOff::On : AutoOnOff::Off);
```